src/libs/opensearch-lib.ts

```diff
@@ -5,9 +5,9 @@ import { OutgoingHttpHeader } from 'http';
 import axios from 'axios';
```

Comment:
If we can, I'd love to remove this file from the diff for this PR. It doesn't seem in-scope and it's making mostly spacing changes, but we should be keeping PRs as focused as possible. We can lint the app and do one PR for these kinds of things if needed.

Everything we need to roll back is in this commit: https://github.com/Enterprise-CMCS/macpro-mako/pull/239/commits/93625aa1912645a6c8c19ab4afd2546dea8ebf44

Should be an easy roll-back.

Reply:
This has to be bundled due to a build error :(